test_chan/test_chan/test_chan.py

```diff
@@ -123,7 +123,7 @@ def test_simple_connection(self):
         assert message == body["message"]
         assert "sender" in body
 
-    def test_full_exchange(self):
+    def terst_full_exchange(self):
```

Comment:
terst? 🤔

Reply:
hey, I had a fever. (And I wanted to just run one test to fix the group issue)

Thanks!